pos_timeout/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2017 - Today: GRAP (http://www.grap.coop)
+# Copyright (C) 2017 - Today: Akretion (http://www.akretion.com)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Point of Sale - timeout",
+    "summary": "Set the timeout of the point of sale",
+    "version": "12.0.1.0.1",
+    "category": "Sales/Point Of Sale",
+    "website": "https://odoo-community.org/",
```

Comment:
website -> https://github.com/OCA/pos/

Reply:
well, in fact it's a backport, so it should be done upstream...

But well, I changed the value.

thanks for your review.

Reply:
:+1: